Lib/test/test_os.py

```diff
@@ -2859,6 +2860,49 @@ def test_getfinalpathname_handles(self):
 
         self.assertEqual(0, handle_delta)
 
+    @support.requires_subprocess()
```

Comment:
Looks like this is a problem on 3.10 as well. I don't recall whether it's in a different place or just not here yet, sorry, you'll need to check.

Reply:
Seems like `os_helper` should have been used instead.

Reply:
Update: I think `requires_subprocess` was added in 3.11, so I'll remove this line as I did for 3.9. (Couldn't find it anywhere in the codebase.)